src/Blueprint.h

```diff
@@ -357,10 +340,18 @@ namespace snowcrash {
             UndefinedElement = 0, // Unknown
             CategoryElement,      // Group of other elements
             CopyElement,          // Human readable text
+            AssetElement,         // Asset of API description
             ResourceElement,      // Resource
             DataStructureElement  // Data Structure
         };
 
+        /** Role of an element */
+        enum Role {
```

Comment:
I am thinking that we might want to use this `Role` in other places too. For example, `Payload` can have `role` as `request` or `response`.